crates/re_data_store/src/store_db.rs

```diff
@@ -73,7 +73,26 @@ impl EntityDb {
 
         // TODO(cmc): batch all of this
         for row in table.to_rows() {
-            self.try_add_data_row(&row)?;
+            self.register_entity_path(&row.entity_path);
+
+            // Look for a `ClearSettings` component, and if it's there, go through the clear path
+            // instead.
+            //
+            // TODO(#3287): If the row contains `ClearSettings`, then we make the arbitrary
+            // decision that everything else on that row is void.
```

Comment:
Is this decision necessary? What's the downside of just also calling `self.try_add_data_row(&row)?` whether the row contains a ClearSettings component or not.

It doesn't seem there is any harm in the ClearSettings component showing up in the timeline. No views should do anything with it so it should otherwise just be ignored.

Reply:
I like it! The only issue is that the clear-related components get cleared themselves which looks pretty weird/confusing in the UI... but with a bit of extra logic in `add_path_op` this now looks quite nice.

![image](https://github.com/rerun-io/rerun/assets/2910679/a751a7c4-b6d0-4050-a078-3264c314de17)